### ISSUE TITLE
support 5-column layouts

### ISSUE THREAD

**Issue by andrewmallis** [kalamuna/kalastatic#46]
https://github.com/kalamuna/mtc/blob/master/src/styles/base/_five_col.scss

let's roll it in?

**Comment by madeofpeople** [comment]
So, I did this in pinnacle just by setting width to the item to 20% which worked.

I also think we need to be a little careful here since there arent "five column layouts" since they are all responsive layouts.

I've been doing this in the sass.

```
%tout-quadriptych {
  @include make-xs-column(12);
  @include make-sm-column(6);
  @include make-md-column(3);
  @include make-lg-column(3);
  @extend %responsive-item;
}
```

**Comment by soniktrooth** [comment]
@madeofpeople, this is how I did it over on MTC: 

```
.col-xs-5th,
.col-sm-5th,
.col-md-5th,
.col-lg-5th {
    position: relative;
    min-height: 1px;
    padding-right: 15px;
    padding-left: 15px;
}

.col-xs-5th {
    width: 20%;
    float: left;
}
@media (min-width: 768px) {
.col-sm-5th {
        width: 20%;
        float: left;
    }
}
@media (min-width: 992px) {
    .col-md-5th {
        width: 20%;
        float: left;
    }
}
@media (min-width: 1200px) {
    .col-lg-5th {
        width: 20%;
        float: left;
    }
}
```

Is your way better?

**Comment by soniktrooth** [comment]
I don't really have a good grasp on what the %balh {} does. Is it some sort of mixin? Can't see it referenced in the sass docs anywhere.

**Comment by madeofpeople** [comment]
So % things are "placeholder selectors"

Here's a good primer:
http://thesassway.com/intermediate/understanding-placeholder-selectors

I may overuse them.
You can @extend them like you can regular classes.

They only get compiled if you use them which is nice (so they dont beef up your final CSS if you don't need the styles, Bootstrap would be wise to use them more).

People abuse Mixins a fair bit... for the following reasons:
Many mixins, take no arguments... (if youre not passing arguments, you shouldnt be using a mixin)... create a placeholder selector (or a regular selector instead)

***BECAUSE***
Every time you include a mixin, it creates a separate declaration… as opposed to:
```
.style1,
.style2 {
…
}
```

Whereas using placeholder selectors does group them.

In conclusion: they are more-efficient argument-lass mixinss that dont end up in your css if you dont use them.

**Comment by madeofpeople** [comment]
Which is why i use
@extend .col-sm-4 
as opposed to @include make-col-sm(4)

(waaaaaay less compiled css comes out)

**Comment by soniktrooth** [comment]
Ok, I've had a bit of a play with this and came up with this: 
```
%tout-quinriptych {
  @include make-xs-column(12);
  @include make-sm-column(2.4);
  @include make-md-column(2.4);
  @include make-lg-column(2.4);
  @extend %responsive-item;
}
```

Seems way easier than I had previously thought and def couldn't find anyone else talking about this on the googles. 

The make-XX-column() functions are just blindly diving the passed value by the amount of columns so `5 ÷ 12 = 2.4`. 

Can anyone see a problem with this? I tried it out and it seems to hold together fine.

**Comment by madeofpeople** [comment]
Seems a reasonable solution to me.
Although it would be pentaptych

**Comment by soniktrooth** [comment]
Dammit—I thought I was being smart. Pentaptych it is.

**Comment by madeofpeople** [comment]
I blame the greeks (again)

**Comment by soniktrooth** [comment]
Pull request #63
